benches/generators.rs

```diff
@@ -27,8 +28,8 @@ use rand::prelude::*;
 use rand::rngs::adapter::ReseedingRng;
 use rand::rngs::OsRng;
 use rand_isaac::{IsaacRng, Isaac64Rng};
-use rand_chacha::{ChaCha8Rng, ChaCha12Rng, ChaCha20Rng};
-use rand_hc::{Hc128Rng, Hc128Core};
+use rand_chacha::*;
```

Comment:
Why don't you use explicit imports?

Reply:
Because this saves time and I don't think it causes any issues?

Reply:
There are several downsides:
* It's harder to tell where names are bound.
* If new items are added, they might clash with existing items.

Not sure how important those are though.